README.md

```diff
@@ -1 +1,29 @@
-"# project24" 
+
+##internet-shop
+Internet-shop with basic functions. 
+According to N-tier architecture project contain several layers:
+* DB
+* DAO
+* Services
+* Controllers
+* Views
+##main functions
+###for Admin: 
+* Manage products
+* Manage users
+* View and delete orders
+###for User:
+* Registration
+* Login/Logout
+* Buy products
+* View Shopping Cart
+* Create orders
+* View own orders and its details
+##technologies 
+Java, Servlets, RBAC Filters, Tomcat, JSP, JSTL, JDBC, MySQL
+ 
+##to run project
+1. Set up MySql DB and Workbench
+2. Run the sql-code from init_db.sql - placed in 'resources' package
+3. Use your properties in ConnectionUtil class
+4. Configure Tomcat
```

Comment:
```suggestion
##to run the project
1. Set up MySql DB and Workbench on your machine
2. Run the SQL-code from the _init_db.sql_ file located in the _resources_ package
3. Insert your MySQL Workbench login and password in the properties in the ConnectionUtil class
4. Configure Tomcat
```
I'd also mention the need to 'inject data' when the website is up and running and how one can try the website out as a user and an admin (e.g. you could say which login and password you've set for the admin)

Reply:
Done